packages/shared/src/utils/constants/jurisdiction.ts

```diff
@@ -0,0 +1,7 @@
+export const Jurisdiction = Object.freeze({
```

Comment:
Just one small concern regarding the name of the file, change it to `jurisdictions-config` in case we are going to add new config related to jurisdictions to it later on.

Reply:
Sure 👌